02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,57 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>About Staci</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <header id="name">Staci Holmes-Segarra</header>
+    <p style= "font-size:30px; text-align: center">San Antonio, Texas</p>
+  <div>
+    <img src="https://scontent-lax3-1.xx.fbcdn.net/v/t1.0-1/p160x160/10357598_1647238348855062_2668382716167648205_n.jpg?oh=1ec04676b5898321d00265b8e4398f66&oe=58200377">
+  </div>
+  <div class="bio">
+    <h2>Biography</h2>
+      <p>I grew up a military brat. My dad was an officer in
+        the U.S. Air Force.  I had to move around to many different places.
+        I had to say good-bye to many friends, but, luckily made new ones fast.
+        As a kid, I didn't appreciate how fortunate I was to live in all these places.
+        Many people don't even have a chance to leave their hometown.  I also think it has
+        helped me to become a better person cause I'm good at change and adapting to new situations.
+       I also can handle a wide range of personalitites.  It's amazing how the different people can be
+     depending on what region of the country they are from. I'm glad I've had the opportunity to live in
+   such a variety of places.</p>
+  </div>
+  <div class="facts">
+    <h3>Places I've Lived<h3>
+      <ul style="list-style-type:none">
+        <a href="http://www.daytoncvb.com/">
+          <li>Dayton, OH</li>
+        </a>
+        <a href="https://en.wikipedia.org/wiki/Ipswich">
+          <li>Ipswich, Suffolk, England</li>
+        </a>
+        <a href="http://www.virginia.org/Cities/chantilly/">
+          <li>Chantilly, Virginia</li>
+        <a>
+        <a href="http://www.cityoftacoma.org/about_tacoma">
+          <li>Tacoma, WA</li>
+        </a>
+        <a href="https://en.wikipedia.org/wiki/Travis_Air_Force_Base">
+          <li>Travis A.F.B., CA</li>
+        </a>
+        <a href="https://en.wikipedia.org/wiki/San_Antonio">
+          <li>San Antonio, TX</li>
+        </a>
+      </ul>
+    </div>
+    <div class="facts">
+      <h3>Random Facts</h3>
+        <ul style="list-style-type:none">
+          <li>Was an Episcopal lay minister</li>
+          <li>I have three girls</li>
+          <li>Am allergic to dogs, but have two</li>
+        </ul>
```

Comment:
No contact info?
How will potential employers contact you if they are interested? :simple_smile:
Great job otherwise.